### PR TITLE
Added Teleport to Selected Blip and associated hotkey.

### PR DESCRIPTION
**Pull request by gir489returns**
Refactored GET_PED_CONFIG_FLAG to use an enum from FiveM.

### CODE REVIEW

File: src/gta/enums.hpp [hunk 1]
```diff
@@ -2032,4 +2023,467 @@ enum class GameMode : int32_t
...
+	CPED_CONFIG_FLAG_FiresDummyRockets                                  = 449,
+	CPED_CONFIG_FLAG_PedIsArresting                                     = 450,
+	CPED_CONFIG_FLAG_IsDecoyPed                                         = 451,
+	CPED_CONFIG_FLAG_HasEstablishedDecoy                                = 452,
+	CPED_CONFIG_FLAG_BlockDispatchedHelicoptersFromLanding              = 453,
+	CPED_CONFIG_FLAG_DontCryForHelpOnStun                               = 454,
+	_CPED_CONFIG_FLAG_0xB68D3EAB                                        = 455,
+	CPED_CONFIG_FLAG_CanBeIncapacitated                                 = 456,
+	_CPED_CONFIG_FLAG_0x4BD5EBAD                                        = 457,
+	CPED_CONFIG_FLAG_DontChangeTargetFromMelee                          = 458,
```
Comment (Yimura) [inline src/gta/enums.hpp]: We've previously opted to not push these because of their shady origin, please remove these.

File: src/util/blip.hpp [hunk 2]
```diff
@@ -45,4 +48,21 @@ namespace big::blip
...
+
+	inline rage::Blip_t* get_selected_blip()
+	{
+		for (int i = 0; i < 1500; i++)
+		{
+
+			auto blip = g_pointers->m_gta.m_blip_list->m_Blips[i].m_pBlip;
+			if (blip)
+			{
+				if (blip->m_display_bits & BlipDisplayBits::BlipIsSelected)
```
Comment (Yimura) [inline src/util/blip.hpp]: Can be combined into a single line 😝

**Review by Yimura — CHANGES_REQUESTED** [review]

File: src/util/blip.hpp (hunk 2 above)
Comment (gir489returns) [reply src/util/blip.hpp]: Done.

File: src/gta/enums.hpp (hunk 1 above)
Comment (gir489returns) [reply src/gta/enums.hpp]: That sucks. :(

**Review by Yimura — APPROVED** [review]

**Pull request merged**